src/tablecloth/time/api/converters.clj

```diff
@@ -20,6 +22,11 @@
 (defn year->local-date [^Year year]
   (-> year (.atMonthDay (java.time.MonthDay/parse "--01-01"))))
 
+(defn year-quarter->local-date [year-quarter format-pattern]
```

Comment:
@rsiddharthan let's make the `format-pattern` optional, and by default support the string format that `org.threeten.extra.YearQuarter/parse` accepts. 

In terms of the location of this function, I think it can make sense to include it in this namespace, but maybe put it at the end with a comment explaining that it (and presumably its partner `year-month->local-date`) are special helpers to help us work with these special types?

Reply:
moving format-pattern as optional is simple to do, and can default to threeten conventions.